lib/utils.js

```diff
@@ -347,6 +347,22 @@ api.validateCapabilityChain = ({capability, capabilityChain}) => {
   return {valid: true};
 };
 
+/**
+ * A summary of the capability to inspect.
+ * @typedef {Object} CapabilitySummary
+ * @property {string} id - The ID of the capability.
+ * @property {string} delegator - The delegator of the capability.
+ * @property {string} invoker - The invoker of the capability.
+ * @property {string} target - The target of the capability.
+ */
```

Comment:
-1 to this. We should just return the full capability. That some use cases may only require these fields is not relevant to  all use cases. Some poor frustrated user may require the entire capability and will need to send a PR to change this. Furthermore, there would then be an incompatibility issue because, for example, `target` doesn't match the data model. Let's not make a new "summary" data model and just return the full capabilities.

Reply:
@dlongley just added a commit that updated the docs.  What do you think of that description?

The target is already computed and there is no reason for the `inspectCapabilityChain` handlert to have to recompute this value. (and of course possibly miscompute)

Reply:
This isn't a problem with the description, it's that we're making assumptions about the calling system that we can't make. They should be given whole capability; we shouldn't be inventing something new that increases their cognitive load/potentially messes up their software stack. Their stack understands zcaps -- we should give them those, not a new thing we invent that happens to be optimized for our particular use case.

Reply:
To be clear, the `capabilityChain` should include all of the zcaps (as zcaps, nothing we invent that is optimized for our use case). It's fine if we want to include other information along side that (things we may have precomputed that others might find helpful).